### DISCUSSION TITLE
Autenticate and limit max viewers.

### DISCUSSION THREAD

**Discussion by TagorePde** [OvenMediaLabs/OvenMediaEngine#1687] [Help]
Can i autenticate users (viewers) using user and password? or token? if so, how?
and also, can i limit how many streams can one user/viewer open?
Regards.

**Comment by bchah** [comment]
[AdmissionWebhooks](https://airensoft.gitbook.io/ovenmediaengine/access-control/admission-webhooks) is a good way to accomplish this, by feeding various events to your own logic.

**Reply by TagorePde** [reply]
Hi, thanks for the reply, there is more info/docs on that? 
is there some api call that i can do and see how many viewers have one stream?
i dont see how can do what i m trying to do.

**Reply by bchah** [reply]
The link above (click the word AdmissionWebhooks) contains all of the documentation. The OME API can do everything you need and more including viewer statistics, so please take a few minutes to review it and you will be much more familiar with the different capabilities. Here's the one that gives you viewer stats: https://airensoft.gitbook.io/ovenmediaengine/rest-api/v1/statistics/current
